pkg/reconciler/serverlessservice/serverlessservice.go

```diff
@@ -124,6 +124,59 @@ func (r *reconciler) reconcilePublicService(ctx context.Context, sks *netv1alpha
 	return nil
 }
 
+// subsetEndpoints computes a subset of all endpoints of size `n` using a consistent
+// selection algorithm. For non empty input, subsetEndpoints returns a copy of the
+// input with the irrelevant endpoints and empty subsets filtered out, if the input
+// size is larger than `n`,
+// Otherwise the input is returned as is.
+func subsetEndpoints(eps *corev1.Endpoints, target string, n int) *corev1.Endpoints {
```

Comment:
Could you describe what `target` is used for?

Reply:
Done.